synapseclient/core/upload/multipart_upload.py

```diff
@@ -410,6 +415,18 @@ def _get_data_chunk(data, part_number, chunk_size):
     return data[(part_number - 1) * chunk_size: part_number * chunk_size]
 
 
+def _get_part_size(part_size, file_size):
+    part_size = part_size or DEFAULT_PART_SIZE
+
+    # can't exceed the maximum allowed num parts
+    part_size = max(
+        part_size,
+        MIN_PART_SIZE,
+        int(math.ceil(file_size / MAX_NUMBER_OF_PARTS))
```

Comment:
test changing default part size for copy to 100MB

Reply:
Added a test for this, although it is skipped in normal testing since it is slow-ish.